src/main/java/gof/strategy/services/ScheduledEventProcessor.java

```diff
@@ -1,12 +1,30 @@
 package gof.strategy.services;
 
 import exception.ExerciseNotCompletedException;
+import gof.strategy.domain.EventType;
 import gof.strategy.domain.ScheduledEvent;
+import gof.strategy.repository.CampaignRepository;
+import gof.strategy.repository.ScheduledEventRepository;
+import gof.strategy.services.strategy.CampaignBannerUpdateStrategy;
+import gof.strategy.services.strategy.CampaignStatusUpdateStrategy;
+import gof.strategy.services.strategy.EventStrategy;
 
 public class ScheduledEventProcessor {
 
+    private EventStrategy campaignBannerUpdateStrategy;
```

Comment:
pls take a look at implementation from this pr for this class
https://github.com/khshanovskyi/KT_tasks/pull/7/files